### DISCUSSION TITLE
(6 kyu) Split Strings

### DISCUSSION THREAD

**Discussion by k0na1bl4** [codewars/codewars.com#2570] [Q&A]
In Visual Studio 2022, my code works, but it throws an error on the codewars website. Please help :(

![codewars](https://user-images.githubusercontent.com/97803473/149633351-fba2c9d2-2fb5-4eee-ac94-6a69fba4e975.png)

**Comment by hobovsky** [comment]
Kata specific questions should be asked in kata discourse: https://www.codewars.com/kata/515de9ae9dcfc28eb6000001/discuss/

If you need some quick help, you can ask at Codewars Discord `#help-solve` channel.

About your problem: `Char` structure is provided by namespace `System`, so you are most probably missing `using System` somewhere.

If you have any follow up questions,  please ask in kata discourse, or in chat.
